ezcontent_publish.permissions.yml

```diff
@@ -1,3 +1,6 @@
 ezcontent publish view user profile:
   title: 'View user profile'
   description: 'Allow user to view profile tab.'
+ezcontent user based shortcut:
```

Comment:
Please see if this is the requirement here, i see second comment(**Permission for roles to enable automatic user shortcut generation on user login.**) says we need a permission that would decide if the default shortcut would be added or not, here you are not creating shortcut itself.

- Accordingly we would need to change the title and description
- Also we would need to add these new permission for editor role in config.